src/Widgets/TopNavigationWidget/SubComponents/Brand.tsx

```diff
@@ -1,17 +1,26 @@
-import { connect, InPlaceEditingOff, LinkTag, ImageTag } from 'scrivito'
+import {
+  connect,
+  InPlaceEditingOff,
+  LinkTag,
+  ImageTag,
+  Obj,
+  Link,
+} from 'scrivito'
 import { HomepageInstance } from '../../../Objs/Homepage/HomepageObjClass'
 import { alternativeTextForObj } from '../../../utils/alternativeTextForObj'
 
 export const Brand = connect(function Brand({
+  linkTo,
   root,
```

Comment:
* [ ] FYI it looks weird to pass in `root` when the component could ask the SDK for it, e.g.,
```tsx
  const root = Obj.root()
  if (!isHomepage(root)) return null
```